src/main/resources/com/puppycrawl/tools/checkstyle/checks/coding/messages.properties

```diff
@@ -4,7 +4,7 @@ avoid.clone.method=Avoid using clone method.
 avoid.double.brace.init=Avoid double brace initialization.
 avoid.finalizer.method=Avoid using finalizer method.
 constructors.declaration.grouping=Constructors should be grouped together. The last grouped constructor is declared at line ''{0}''.
-covariant.equals=covariant equals without overriding equals(java.lang.Object).
+covariant.equals=Covariant Equals without overriding equals(java.lang.Object).
```

Comment:
please revert this change, it might break compatibility, as there is ability to filterout violations by message.

Reply:
Reverted the change you mentioned. But my branch got diverged somehow, will raise a new PR closing this one.